src/PhpSpec/Listener/CollaboratorMethodNotFoundListener.php

```diff
@@ -110,10 +110,10 @@ public function afterExample(ExampleEvent $event)
     }
 
     /**
-     * @param string $classname
+     * @param mixed string|object $classname
```

Comment:
Is this right? I thought you'd omit `mixed` here

Reply:
hmm indeed there's not much point to it.